### ISSUE TITLE
Documentation for qubes-vm-update is missing

### ISSUE THREAD

**Issue by marmarek** [QubesOS/qubes-issues#9196]
[How to file a helpful issue](https://www.qubes-os.org/doc/issue-tracking/)

### Qubes OS release

R4.2

### Brief summary

There is no man page and no mention in qubes-doc

### Expected behavior

Documentation exists. The man page should:
- describe options
- meaning for specific exit codes

There should be also a mention in qubes-doc at https://www.qubes-os.org/doc/how-to-update/

**Comment by marmarek** [comment]
What is missing here? or maybe it can be closed already?

**Comment by alimirjamali** [comment]
> What is missing here? or maybe it can be closed already?

Yes. The PR was merged last month by @rapenne-s. Here is a link to it:
https://github.com/QubesOS/qubes-doc/pull/1408
